connect/runtime/src/test/java/org/apache/kafka/connect/runtime/standalone/StandaloneHerderTest.java

```diff
@@ -115,64 +115,61 @@ public class StandaloneHerderTest {
     private static final String TOPICS_LIST_STR = "topic1,topic2";
     private static final String WORKER_ID = "localhost:8083";
     private static final String KAFKA_CLUSTER_ID = "I4ZmrWqfT2e-upky_4fdPA";
+    private static final Long WAIT_TIME = 1000L;
```

Comment:
nit: 1 second is probably good enough, but it costs nothing to increase this now.
Also the constant should probably include the unit, so `WAIT_TIME_MS`.
```suggestion
    private static final Long WAIT_TIME = 15000L;
```